### ISSUE TITLE
Odd behavior with math.combinatorics

### ISSUE THREAD

**Issue by sarangak** [borkdude/babashka#370]
**version**

0.0.88

**platform**

OS X 10.15.1

**problem**

When pulling in https://github.com/clojure/math.combinatorics the behavior of the permutations function does not match JVM Clojure.

**repro**

```
$ clojure -Sdeps '{:deps {org.clojure/math.combinatorics {:mvn/version "0.1.6"}}}' -e "(use 'clojure.math.combinatorics) (permutations [:a :b])"
((:a :b) (:b :a))
$ bb --classpath "$(clojure -Spath -Sdeps '{:deps {org.clojure/math.combinatorics {:mvn/version "0.1.6"}}}')" -e "(use 'clojure.math.combinatorics) (permutations [:a :b])"
((:a :b) (:a :a))

$ clojure -Sdeps '{:deps {org.clojure/math.combinatorics {:mvn/version "0.1.6"}}}' -e "(use 'clojure.math.combinatorics) (permutations [:a :b :c])"
((:a :b :c) (:a :c :b) (:b :a :c) (:b :c :a) (:c :a :b) (:c :b :a))
$ bb --classpath "$(clojure -Spath -Sdeps '{:deps {org.clojure/math.combinatorics {:mvn/version "0.1.6"}}}')" -e "(use 'clojure.math.combinatorics) (permutations [:a :b :c])"
((:a :b :c) (:a :b :b) (:a :a :b) (:a :a :a))
```

**expected behavior**

Behavior on both platforms should match.

**Comment by borkdude** [comment]
@sarangak Thanks for spotting this. Should be fixed with https://7167-201467090-gh.circle-artifacts.com/0/release/babashka-0.0.89-SNAPSHOT-macos-amd64.zip. Can you test?

**Comment by borkdude** [comment]
Released as 0.88-2.

**Comment by sarangak** [comment]
Worked for me!
